python/templates/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Language detector</title>
+  <meta charset="UTF-8">
+  <link rel='stylesheet' href='/static/stylesheets/style.css' />
+</head>
+<body>
+  <h1>Language detector</h1>
+  <form action="/" method="get">
+    <input type="text" name="text" aria-label= "Enter text." value="{{ text }}"/>
```

Comment:
```suggestion
    <input type="text" name="text" aria-label= "Enter text" value="{{ text }}"/>
```

Reply:
Hey Pamela -- one other thing. While the stack trace prints out when running in debug mode locally. It doesn't print when running in Azure. Is there an easy way to fix this? In the tutorial, it is possible for someone to forget to issue the commands to set environment variables for the cognitive services account name and key and they could experience an error.

Reply:
Hm, what we've been doing is hooking our apps up to Application Insights, and then the errors turn up in the Failures tab. I thought the stack traces do show up in the Log stream / Kudu logs even if you don't do that, but I might be wrong. I'd have to actually try deploying this one and try it out to see the experience, which I won't be able to do today.

Reply:
Let's take this up on Monday. I don't really have more time to spend on it today either.